Traceback (most recent call last):
  File "/ocean/projects/cis230002p/palavall/nanoGPT/train.py", line 182, in <module>
    model = GPT.from_pretrained(init_from, override_args, cache_dir="/ocean/projects/cis230002p/palavall/nanoGPT/cache/")
TypeError: GPT.from_pretrained() got an unexpected keyword argument 'cache_dir'
